modules/sugar/src/main/ts/ephox/sugar/api/node/Head.ts

```diff
@@ -0,0 +1,10 @@
+import Element from './Element';
+import { Document, HTMLHeadElement } from '@ephox/dom-globals';
+
+export const getHead = (doc: Element<Document>): Element<HTMLHeadElement> => {
+  const b = doc.dom().getElementsByTagName('head')[0];
```

Comment:
Yeah, I only did it this way since I saw it done like this in other places. 

I'm guessing it might be to work around something in a very old browser?

Reply:
Worth noting we use `const b = doc.dom().body;` in the `Body` module, so it'd likely make sense to just use `const h = doc.dom().head;` here :shrug:

Reply:
I'd say the version in TinyMCE is old code from pre-IE 9 days tbh.

Reply:
yep, `.head` has worked since IE9.

Reply:
👍  Done.